Sources/Misc/Deprecations.swift

```diff
@@ -56,6 +56,64 @@ public extension Purchases {
         return await eligiblePromotionalOffers(forProduct: product)
     }
 
+    /**
+     * Enable debug logging. Useful for debugging issues with the lovely team @RevenueCat.
+     */
+    @available(*, deprecated, message: "use Purchases.logLevel instead")
+    @objc static var debugLogsEnabled: Bool {
+        get { logLevel == .debug }
+        set { logLevel = newValue ? .debug : .info }
+    }
+
+    /**
+     * Deprecated
```

Comment:
ooff I guess we missed adding docs for this one (before this PR)

Could we add a link to https://docs.revenuecat.com/docs/user-ids and maybe a comment like 
```suggestion
     * Deprecated: Configure  behavior through the RevenueCat dashboard instead.
     * Determines whether or not users may reuse a subscription from an App or Play Store account that already has that subscription active.
     * ### Related articles: 
     * - https://docs.revenuecat.com/docs/user-ids
```